boards/arm/bcm958402m2_m7/bcm958402m2_m7_defconfig

```diff
@@ -18,3 +18,13 @@ CONFIG_UART_NS16550=y
 # Enable Console
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
+
+# Enable PCIe EP
+CONFIG_PCIE_ENDPOINT=y
+CONFIG_PCIE_EP_IPROC=y
+CONFIG_PCIE_EP_IPROC_V2=y
+
+# Enable PAXDMA
+CONFIG_DMA=y
+CONFIG_DMA_IPROC_PAX_V2=y
```

Comment:
What about DMA_PL330?

Reply:
Will enable in https://github.com/zephyrproject-rtos/zephyr/pull/32316